pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -178,6 +178,11 @@ type LeaderElectionConfiguration struct {
 type KubeletConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
```

Comment:
Technically, you need to bump the API since you are making breaking changes. May be time to move it to `alpha2`? Are you still targeting `beta1` for v1.6?